src/event_form/event-form-data.factory.js

```diff
@@ -537,17 +537,13 @@ function EventFormDataFactory(rx, calendarLabels, moment, OpeningHoursCollection
       if (formData.calendar.calendarType === 'single') {
         formData.openingHours = []; // Major info is created from this property and not the one in calendar
         if (appConfig.calendarHighlight.date) {
-          formData.addTimeSpan(
-              new Date(appConfig.calendarHighlight.date),
-              appConfig.calendarHighlight.startTime || '',
-              appConfig.calendarHighlight.startTime ?
-                  moment(appConfig.calendarHighlight.date + ' ' +
-                      appConfig.calendarHighlight.startTime, 'YYYY-MM-DD HH:mm').toDate() : '',
-              appConfig.calendarHighlight.endTime || '',
-              appConfig.endTime ?
-                  moment(appConfig.calendarHighlight.date + ' ' +
-                      appConfig.calendarHighlight.endTime, 'YYYY-MM-DD HH:mm').toDate() : ''
-          );
+          var startTime =  appConfig.calendarHighlight.startTime ?
+          moment(appConfig.calendarHighlight.date + ' ' + appConfig.calendarHighlight.startTime, 'YYYY-MM-DD HH:mm')
+          .toDate() : '';
+          var endTime =  appConfig.calendarHighlight.endTime ?
+          moment(appConfig.calendarHighlight.date + ' ' + appConfig.calendarHighlight.endTime, 'YYYY-MM-DD HH:mm')
+          .toDate() : '';
+          formData.addTimeSpan(startTime, endTime);
```

Comment:
I don't think the formatting is right. e.g. 2 spaces after `startTime =` 

Maybe you can use prettier to format that selection `Format Selection (⌘K ⌘F) - Format the selected text.`